server/app/api/v3.py

```diff
@@ -175,6 +182,7 @@ def add_resource(resource, url):
     add_resource(ColorsAPI, "/colors")
     # Computation routes
     add_resource(DiffExpObsAPI, "/diffexp/obs")
+    add_resource(DiffExpObs2API, "/diffexp/obs2")
```

Comment:
If anyone has other ideas on a route name, this can be easily changed. Semantics are (mostly) the same as the previous route - it is the encoding that has changed.

Reply:
Maybe we could introduce versioning by route? This file is `v3` but we could add a `register_api_v4` with this route only. This is probably a bit more work so I would keep this as an optional suggestion.

Reply:
I will follow your lead.  The old API is going away at some point in the near future.  If that is best implemented by versioning, then it makes sense to me.  I didn't use that approach as the API is completely incompatible due to the new encoding - ie, it isn't just a revision of the old API.  It is a new API implementing the same functionality.

Reply:
@atolopko-czi what is your preference? If we plan to remove the old route after the release, we can just switch the new one to `obs` (no conflict risk since BE+FE are deployed together) so I'd skip versioning.